### ISSUE TITLE
sqlite requires System.Data.SQLite, doesn't play nicely with SQLite PCL

### ISSUE THREAD

**Issue by ChangePlaces** [CollaboratingPlatypus/PetaPoco#347]
When just the SQLite PCL nuget package is installed, petapoco refuses to create a connection, insisting that the db provider doesn't exist. Maybe instead of hardcoding System.Data.SQLite in the code, it could attempt to create a `new SQLite("path");` ?

**Comment by zhengyangyong** [comment]
yes,also think need support t4 template db first,auto gen poco class
